fps-emit/package.json

```diff
@@ -51,6 +51,7 @@
     "react": "^16.4.2",
     "react-dom": "^16.4.2",
     "react-testing-library": "^5.0.0",
+    "react-scripts": "2.0.3",
```

Comment:
`fps-emit` isn't one of the benchmarks, and so it shouldn't depend on `react-scripts` at all.  Remove this and fix up the lockfile, and we can get it merged in.